### PR TITLE
Dịch "linear-regression-gluon_vn" - Phần 5

### LINKED ISSUE #668: Dịch "linear-regression-gluon_vn" - Phần 5
Phần cần dịch: https://github.com/aivivn/d2l-vn/blame/master/chapter_linear-networks/linear-regression-gluon_vn.md#L209-L254

### PR DESCRIPTION
**Pull request by NhatQuanBui**
## Pull Request Checklist

**Đánh dấu tick (sau khi "Create Pull Request") vào các mục dưới đây**

* [x] Pull Request này tương ứng với issue nào? Trả lời số issue sau dấu `#` (không có dấu cách): Close #668 

* [x] Bản dịch này có bám sát **[Bảng thuật ngữ](https://github.com/aivivn/d2l-vn/blob/master/glossary.md)** không? Nếu một từ/cụm chưa có trong Bảng thuật ngữ, hãy tạo một Pull Request riêng đề xuất cách dịch từ/cụm đó.

* [x] Format của bản gốc có được giữ nguyên không?
    * Các phần in nghiêng, in đậm.
    * Tiêu đề (số lượng dấu `#` đầu dòng).
    * Bảng biểu, chú thích cho bảng (dòng phía trên mỗi bảng bắt đầu bằng dấu `:`).
    * Dịch các từ trong hình vẽ nếu cần. Các dòng có hình có dạng: `caption`.
    * Dịch các chú thích hình vẽ (thay các cụm `*dịch chú thích ảnh phía trên*` bằng bản dịch tiếng Việt).
    * Không dịch các phần code (nằm giữa hai dấu `````).
    * Copy các công thức toán từ bản gốc (các đoạn có `$`).
    * Giữ các dòng gán nhãn (bắt đầu với `:label:`).
    * Không tự thêm bớt các dòng trắng.
     
* [x] Trong một đoạn văn, mỗi câu văn đã được viết trong một dòng, giữa các dòng không có dòng trắng. Markdown sẽ coi những dòng liền nhau không có dòng trắng là một đoạn văn. Việc này giúp công đoạn review được thuận tiện hơn.

* [x] Tất cả chữ trong các hình vẽ đã được dịch. Nếu chưa, vui lòng xem thêm [hướng dẫn hoặc yêu cầu dịch hình](../img/README.md).

* [x] Điền tên của bạn vào mục "Những người thực hiện" ở cuối file.

* [x] **Sau khi được approve**, điền tên những reviewers mà bạn thấy có nhiều đóng góp cho Pull Request này. Xem danh sách Contributors **[tại đây](../docs/contributors_info.md/)**.

## Với reviewer

1. Chỉ review nếu Pull Request này đã sẵn sàng cho việc review.

2. **Bạn có thể đề xuất thay đổi cách dịch mỗi dòng trực tiếp như trong hình dưới đây:**
img

### CODE REVIEW

File: chapter_linear-networks/linear-regression-gluon_vn.md [hunk 1]
```diff
@@ -235,7 +237,7 @@ loss = gloss.L2Loss()  # The squared loss is also known as the L2 norm loss
 ## Defining the Optimization Algorithm
 -->
 
-## *dịch tiêu đề phía trên*
+## Định nghĩa thuật toán tối ưu hóa
```
Comment (honghanhh) [inline chapter_linear-networks/linear-regression-gluon_vn.md]:
```suggestion
## Định nghĩa Thuật toán tối ưu hóa
```

File: chapter_linear-networks/linear-regression-gluon_vn.md [hunk 2]
```diff
@@ -212,15 +212,17 @@ Just be careful to remember that since the parameters have not been initialized
 ## Defining the Loss Function
 -->
 
-## *dịch tiêu đề phía trên*
+## Định nghĩa hàm mất mát
```
Comment (honghanhh) [inline chapter_linear-networks/linear-regression-gluon_vn.md]: Em cảm ơn bản dịch của anh ạ. Em có 1 chút comment nho nhỏ ạ ở dưới ạ

Comment (honghanhh) [inline chapter_linear-networks/linear-regression-gluon_vn.md]:
```suggestion
## Định nghĩa Hàm mất mát
```

File: chapter_linear-networks/linear-regression-gluon_vn.md [hunk 3]
```diff
@@ -244,7 +246,9 @@ the optimization algorithm we wish to use (`sgd`), and a dictionary of hyper-par
 SGD just requires that we set the value `learning_rate`, (here we set it to 0.03).
 -->
 
-*dịch đoạn phía trên*
+Minibatch SGD và các biến thể liên quan đều là các công cụ chuẩn cho việc tối ưu hóa những mạng thần kinh và vì vậy Gluon hỗ trợ SGD cùng với một số biến thể của thuật toán này thông qua lớp 'Trainers' của nó.
```
Comment (honghanhh) [inline chapter_linear-networks/linear-regression-gluon_vn.md]:
```suggestion
Minibatch SGD và các biến thể liên quan đều là các công cụ chuẩn cho việc tối ưu hóa những mạng nơ-ron và vì vậy Gluon hỗ trợ SGD cùng với một số biến thể của thuật toán này thông qua lớp 'Trainers' của nó.
```

File: chapter_linear-networks/linear-regression-gluon_vn.md [hunk 4]
```diff
@@ -244,7 +246,9 @@ the optimization algorithm we wish to use (`sgd`), and a dictionary of hyper-par
...
+Khi chúng ta khởi tạo lớp 'Trainer', chúng ta sẽ chỉ định các tham số để tối ưu hóa ( có thể lấy từ các mạng của chúng tôi thông qua `net.collect_params()`), thuật toán tối ưu hóa mà chúng tôi muốn sử dụng (`sgd`), và một từ điển siêu tham số theo yêu cầu bởi thuật toán tối ưu hóa của chúng tôi.
```
Comment (honghanhh) [inline chapter_linear-networks/linear-regression-gluon_vn.md]:
```suggestion
Khi chúng ta khởi tạo lớp 'Trainer', chúng ta sẽ chỉ định các tham số để tối ưu hóa (có thể lấy từ các mạng của chúng ta thông qua `net.collect_params()`), thuật toán tối ưu hóa mà chúng ta muốn sử dụng (`sgd`), và một từ điển siêu tham số theo yêu cầu bởi thuật toán tối ưu hóa của chúng ta.
```

File: chapter_linear-networks/linear-regression-gluon_vn.md [hunk 5]
```diff
@@ -212,15 +212,17 @@ Just be careful to remember that since the parameters have not been initialized
...
+## Định nghĩa Hàm mất mát
 
 <!--
 In Gluon, the `loss` module defines various loss functions.
 We will use the imported module `loss` with the pseudonym `gloss`, to avoid confusing it for the variable holding our chosen loss function.
 In this example, we will use the Gluon implementation of squared loss (`L2Loss`).
 -->
 
-*dịch đoạn phía trên*
+Trong Gluon, mô-đun 'loss' định nghĩa các hàm mất mát khác nhau.
```
Comment (duythanhvn) [inline chapter_linear-networks/linear-regression-gluon_vn.md]:
```suggestion
Trong Gluon, mô-đun `loss` định nghĩa các hàm mất mát khác nhau.
```

File: chapter_linear-networks/linear-regression-gluon_vn.md [hunk 6]
```diff
@@ -212,15 +212,17 @@ Just be careful to remember that since the parameters have not been initialized
...
+Chúng ta sẽ sử dụng mô-đun 'loss' được thêm vào với biệt hiệu là 'gloss', để tránh nhầm lẫn nó cho các biến giữ lại các hàm mất mát của chúng ta đã chọn.
```
Comment (duythanhvn) [inline chapter_linear-networks/linear-regression-gluon_vn.md]:
```suggestion
Chúng ta sẽ sử dụng mô-đun `loss` được thêm vào gọi là `gloss`, để tránh nhầm lẫn nó với các biến giữ lại hàm mất mát mà ta đã chọn.
```

File: chapter_linear-networks/linear-regression-gluon_vn.md [hunk 7]
```diff
@@ -212,15 +212,17 @@ Just be careful to remember that since the parameters have not been initialized
...
+Trong ví dụ này, chúng ta sẽ sử dụng triển khai Gluon của mất mát bình phương (`L2Loss`).
```
Comment (duythanhvn) [inline chapter_linear-networks/linear-regression-gluon_vn.md]: Cụm ```(`L2Loss`)``` này thì giữ lại format, mà mấy từ trên kia lại không giữ format về dấu @@

File: chapter_linear-networks/linear-regression-gluon_vn.md [hunk 8]
```diff
@@ -235,7 +237,7 @@ loss = gloss.L2Loss()  # The squared loss is also known as the L2 norm loss
 ## Defining the Optimization Algorithm
 -->
 
-## *dịch tiêu đề phía trên*
+## Định nghĩa Thuật toán tối ưu hóa
```
Comment (duythanhvn) [inline chapter_linear-networks/linear-regression-gluon_vn.md]:
```suggestion
## Định nghĩa Thuật toán Tối ưu hóa
```

File: chapter_linear-networks/linear-regression-gluon_vn.md [hunk 9]
```diff
@@ -244,7 +246,9 @@ the optimization algorithm we wish to use (`sgd`), and a dictionary of hyper-par
 SGD just requires that we set the value `learning_rate`, (here we set it to 0.03).
 -->
 
-*dịch đoạn phía trên*
+Minibatch SGD và các biến thể liên quan đều là các công cụ chuẩn cho việc tối ưu hóa những mạng nơ-ron và vì vậy Gluon hỗ trợ SGD cùng với một số biến thể của thuật toán này thông qua lớp 'Trainers' của nó.
```
Comment (duythanhvn) [inline chapter_linear-networks/linear-regression-gluon_vn.md]:
```suggestion
Minibatch SGD và các biến thể liên quan đều là các công cụ chuẩn cho việc tối ưu hóa những mạng nơ-ron và vì vậy Gluon hỗ trợ SGD cùng với một số biến thể của thuật toán này thông qua lớp `Trainers` của nó.
```

File: chapter_linear-networks/linear-regression-gluon_vn.md [hunk 10]
```diff
@@ -244,7 +246,9 @@ the optimization algorithm we wish to use (`sgd`), and a dictionary of hyper-par
...
+Khi chúng ta khởi tạo lớp 'Trainer', chúng ta sẽ chỉ định các tham số để tối ưu hóa (có thể lấy từ các mạng của chúng ta thông qua `net.collect_params()`), thuật toán tối ưu hóa mà chúng ta muốn sử dụng (`sgd`), và một từ điển siêu tham số theo yêu cầu bởi thuật toán tối ưu hóa của chúng ta.
```
Comment (duythanhvn) [inline chapter_linear-networks/linear-regression-gluon_vn.md]:
```suggestion
Khi chúng ta khởi tạo lớp `Trainer`, ta sẽ chỉ định các tham số để tối ưu hóa (có thể lấy từ các mạng của chúng ta thông qua `net.collect_params()`), thuật toán tối ưu hóa mà chúng ta muốn sử dụng (`sgd`), và một từ điển siêu tham số theo yêu cầu bởi thuật toán tối ưu hóa của chúng ta.
```

File: chapter_linear-networks/linear-regression-gluon_vn.md [hunk 11]
```diff
@@ -212,15 +212,17 @@ Just be careful to remember that since the parameters have not been initialized
...
 
 <!--
 In Gluon, the `loss` module defines various loss functions.
 We will use the imported module `loss` with the pseudonym `gloss`, to avoid confusing it for the variable holding our chosen loss function.
 In this example, we will use the Gluon implementation of squared loss (`L2Loss`).
 -->
 
-*dịch đoạn phía trên*
+Trong Gluon, mô-đun `loss` định nghĩa các hàm mất mát khác nhau.
+Chúng ta sẽ sử dụng mô-đun `loss` được thêm vào gọi là `gloss`, để tránh nhầm lẫn nó với các biến giữ lại hàm mất mát mà ta đã chọn.
```
Comment (lkhphuc) [inline chapter_linear-networks/linear-regression-gluon_vn.md]:
```suggestion
Chúng ta sẽ sử dụng mô-đun `loss` được thêm vào dưới tên gọi là `gloss`, để tránh nhầm lẫn nó với biến đang giữ hàm mất mát mà ta đã chọn.
```
